samples/openthread/ncp/README.rst

```diff
@@ -0,0 +1,264 @@
+.. _ot_ncp_sample:
+
+Thread: NCP
+###########
+
+The Thread NCP sample demonstrates the usage of OpenThread Network Co-Processor architecture inside the Zephyr shell.
```

Comment:
I think we should use something like "Zephyr environment", Zephyr's shell is not enabled here

Reply:
Done